outlines/models/__init__.py

```diff
@@ -19,6 +20,3 @@
 from .vllm import VLLM, vllm
 
 LogitsGenerator = Union[Transformers, LlamaCpp, OpenAI, ExLlamaV2Model, MLXLM, VLLM]
-
-LocalModel = LlamaCpp
-APIModel = Union[AzureOpenAI, OpenAI, Anthropic, Gemini]
```

Comment:
I would keep these types instead of using a class attribute to distinguish between API and local models.